app/views/manage_assessments/outcome_coverages/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= simple_form_for @outcome_coverage, url: manage_assessments_course_outcome_coverages_path(course_id: @outcome_coverage.course.id) do |form| %>
```

Comment:
I think we can just do: `manage_assessments_course_outcome_coverages_path(@outcome_coverage.course)`